modules/auxiliary/scanner/nfs/nfsmount.rb

```diff
@@ -10,74 +10,101 @@ class MetasploitModule < Msf::Auxiliary
 
   def initialize
     super(
-      'Name'          => 'NFS Mount Scanner',
-      'Description'   => %q{
+      'Name' => 'NFS Mount Scanner',
+      'Description' => %q{
         This module scans NFS mounts and their permissions.
       },
-      'Author'	       => ['<tebo[at]attackresearch.com>'],
-      'References'     =>
-        [
-          ['CVE', '1999-0170'],
-          ['URL',	'https://www.ietf.org/rfc/rfc1094.txt']
-        ],
+      'Author'	=> ['<tebo[at]attackresearch.com>'],
+      'References' => [
+        ['CVE', '1999-0170'],
+        ['URL',	'https://www.ietf.org/rfc/rfc1094.txt']
```

Comment:
I'm not sure how this cheeky little tab ended up here. Nor how the tests passed in its presence.

```suggestion
        ['URL', 'https://www.ietf.org/rfc/rfc1094.txt']
```